frontend/src/components/BookingForm.js

```diff
@@ -50,8 +59,8 @@ const BookingForm = () => {
     let availabilities = new Map()
 
     existingBookings.forEach((booking) => {
-      let currDate = new Date(getFormattedDateString(booking.checkIn))
-      const endDate = new Date(getFormattedDateString(booking.checkOut))
+      let currDate = new Date(getFormattedDateString(booking.check_in))
```

Comment:
Unrelated to this PR but do we need dayjs? The `booking.check_in` is a Firestore [timestamp](https://firebase.google.com/docs/reference/js/v8/firebase.firestore.Timestamp) which has a `toDate` method itself. Just a comment for other maintainers.

Reply:
Iirc, I think I added it in because I was getting some runtime errors due to incompatible types between the booking form state and the selectable dates of the MUI date picker.

But if there is a better way to do it, I think it would be a good idea to look into refactoring a little bit to get rid of dayjs if it's not absolutely needed.

Reply:
If mui depends on it, then we can just leave it as is.